### PR TITLE
Update Seed File

### LINKED ISSUE #6: Setup Test Suite
- [x] RSpec
- [x] Factory Bot
- [x] Create seed file

### PR DESCRIPTION
**Pull request by lpile**
## Changes proposed in this pull request:
* Update seed file
* Add `dependent: :destroy` to User model
* Update user factories

## Card(s) closed in this pull request:
close #6 

## What did you struggle on to complete?
* Nothing at this moment.

### Test Coverage Percentage: 100%
- [x] Tested my new feature(s) as well as any feasible edge cases (if possible)
- [x] Checked coverage/index.html - did not add any new code that's not covered by testing (if possible)
- [x] Ran the test suite - all tests are passing
- [x] Merged in the latest master to my branch with git pull origin master & resolved merge conflicts
- [x] I have commented my code, particularly in hard-to-understand areas

### CODE REVIEW

File: app/models/user.rb [hunk 1]
```diff
@@ -1,7 +1,7 @@
 class User < ApplicationRecord
-  has_many :user_events
+  has_many :user_events, dependent: :destroy
```
Comment (annekemcgrady) [inline app/models/user.rb]: I'm interested to learn about what dependent: :destroy is doing here.

Comment (lpile) [reply app/models/user.rb]: By adding the `dependent: :destroy` option to the has_many :through associations, the User record will be removed when UserEvents row is destroyed.

But I just realized that I am using it wrong here and will submit a PR fix shortly.
